add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -556,6 +556,7 @@ var LoopRoomsInternal = {
       for (let room of roomsList) {
         // See if we already have this room in our cache.
         let orig = this.rooms.get(room.roomToken);
+        // let orig = extend({}, this.rooms.get(room.roomToken));
```

Comment:
@Standard8, our recollection is that this was something you suggested relating to making sure we have the right version somehow.  Do recall what the point of using the commented-out form (which has some other issue) was?